base/applications/cmdutils/where/where.c

```diff
@@ -0,0 +1,564 @@
+/*
+ * PROJECT:     ReactOS WHERE command
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Search executable files
+ * COPYRIGHT:   Copyright 2021 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+#ifdef __REACTOS__
+    #include <windef.h>
+    #include <winbase.h>
+    #include <winuser.h>
+    #include <winnls.h>
+#else
+    #include <windows.h>
+#endif
```

Comment:
Please remove that and instead just use what's inside the `__REACTOS__`. You are writing a tool for ReactOS, not for somewhere else.

Reply:
`miniconutils.h` is deleted.